eip/EIPS/eip-4337.md

```diff
@@ -98,12 +105,14 @@ A Wallet that supports aggregated signature should have the interface:
 ```solidity
 interface IAggregatedWallet is IWallet {
   function getAggregator() view returns (address);
-  function getPublicKey() view returns (bytes memory);
 }
 ```
 * **getAggregator()** returns the aggregator this wallet supports.
 * The wallet is guaranteed to that the EntryPoint will only call its `validateUserOp` (with no signature) after calling the aggregator's validateSignatures
-* **getPublicKey()** is called by the aggregator, to get the wallet-specific public key.
+* The wallet should also support aggregator-specific getter (e.g. `getAggregationInfo()`).
```

Comment:
Why can't an aggregator be a "black box" (before optimization)?  After the data from all the wallets is collected, why can't it use a pure function in the aggregator to aggregate it?  I can see why it'll be optimized for common aggregators, but if someone builds an experimental aggregator it could work without waiting for bundlers to add support.